both/lib/methods.js

```diff
@@ -23,6 +23,35 @@ Meteor.methods({
       }
     });
   },
+  registerJobInterest: function(jobId) {
+    check(jobId, String);
+
+    var job = Jobs.findOne({
+      _id: jobId
+    });
+
+    if (!job)
+      throw new Meteor.Error("Could not find job.");
+
+    if (job.status !== "active")
```

Comment:
make sure to disable the interested in button for non-active jobs in the UI also - prevent this error from occuring in the first place - still leave the checking in place though